dev/dev_setup.sh

```diff
@@ -279,13 +287,20 @@ put_brew_in_path() {
 set_ubuntu_language_encoding() {
     case $CRYPTOL_PLATFORM in
         $UBUNTU20 | $UBUNTU22)
-            notice "Adding language environment variables to $VAR_FILE"
-            echo "export LANG=C.UTF-8" >> $VAR_FILE
-            echo "export LC_ALL=C.UTF-8" >> $VAR_FILE;;
-        *) ;;
+            if $LANG!= *"UTF-8"* || $LANG != *"utf-8"*; then
```

Comment:
I don't think this conditional is going to work. Try
```
case x"$LANG" in
    *UTF-8*|*utf-8*) ;;
    *) notice ... ;;
```
(the x makes it not croak if the variable isn't set)